wcqtlib/train/models.py

```diff
@@ -156,7 +156,15 @@ def _build_network(self):
                 raise InvalidNetworkDefinition(
                     "Each layer must contain a 'type'")
 
-            network = layer_class(network, **layer)
+            if object_definition.get("batch_norm", False) is True \
+                    and 'nonlinearity' in layer:
+                network = lasagne.layers.batch_norm(
```

Comment:
I thought lasagne's batch norm automatically does the nonlinearity detection logic for us?  Or am I mis-remembering?

Also, backslashing across lines is to be discouraged: use parentheses instead.